pagopa/pn-frontend
# PN-10516 - Add some analog events on timeline

## Short description
Show on timeline the `RECAG010`, `RECRS010` and `RECRN010` analog events. It also show `RECAG012` event.

## List of changes proposed in this pull request
- Add analog events on timeline

## How to test
`RECAG010` -> XQZH-YURU-ZPQR-202404-J-1 (ADA, DEV), you should see: "Il tentativo di consegna della raccomandata 890 numero {{numero raccomandata}} per {{name}} {{CF}} non è andato a buon fine per destinatario assente."

`RECRS010` -> ZVXL-GNWP-HPRX-202404-G-1 (Cristoforo Colombo, DEV), you should see: "Il tentativo di consegna della raccomandata semplice numero {{numero raccomandata}} non è andato a buon fine per destinatario assente."

`RECRN010` -> EDYE-PKZK-WRJA-202404-D-1 (ADA, DEV), you should see: "Il tentativo di consegna della raccomandata A/R numero {{numero raccomandata}} non è andato a buon fine per destinatario assente."

`RECAG012` -> 	MXJY-VRVM-UPXM-202404-Q-1 (Cristofoto Colombo, DEV), you should see: "La raccomandata 890 numero {{numero raccomandata}} è stata ritirata o ha compiuto la sua giacenza."
